packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -468,6 +471,12 @@ export default class Menu extends Component {
                 ]}
               />
             </Tooltip>
+          ) : this._areProxiesErrors() ? (
+            <Tooltip content={_('someProxiesHaveErrors')}>
+              <span className='text-warning'>
+                <Icon icon='alarm' />
+              </span>
+            </Tooltip>
```

Comment:
I think we should give more info, for instance, the user should be able to tell if there is a new error in case they get use to having an error on a proxy.

Maybe we display the number of proxies that have errors? Maybe in a pill?

Reply:
Yes, I think showing the number of errors is important. Maybe you can use the same icon we use in Home/Host to show warnings next to hosts' names to do that.

And then the user can check the actual errors on the Proxies page, right?